mmv1/third_party/terraform/services/resourcemanager/ephemeral_google_service_account_id_token.go

```diff
@@ -0,0 +1,153 @@
+package resourcemanager
+
+import (
+	"context"
+	"fmt"
+
+	"google.golang.org/api/idtoken"
+	"google.golang.org/api/option"
+
+	"github.com/hashicorp/terraform-plugin-framework-validators/setvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/ephemeral"
+	"github.com/hashicorp/terraform-plugin-framework/ephemeral/schema"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-provider-google/google/fwmodels"
+	"github.com/hashicorp/terraform-provider-google/google/fwtransport"
+	"github.com/hashicorp/terraform-provider-google/google/fwutils"
+	"github.com/hashicorp/terraform-provider-google/google/fwvalidators"
+	"google.golang.org/api/iamcredentials/v1"
+)
+
+var _ ephemeral.EphemeralResource = &googleEphemeralServiceAccountIdToken{}
+
+func GoogleEphemeralServiceAccountIdToken() ephemeral.EphemeralResource {
+	return &googleEphemeralServiceAccountIdToken{}
+}
+
+type googleEphemeralServiceAccountIdToken struct {
+	providerConfig *fwtransport.FrameworkProviderConfig
+}
+
+func (p *googleEphemeralServiceAccountIdToken) Metadata(ctx context.Context, req ephemeral.MetadataRequest, resp *ephemeral.MetadataResponse) {
+	resp.TypeName = req.ProviderTypeName + "_service_account_id_token"
+}
+
+type ephemeralServiceAccountIdTokenModel struct {
+	TargetAudience       types.String `tfsdk:"target_audience"`
+	TargetServiceAccount types.String `tfsdk:"target_service_account"`
+	Delegates            types.Set    `tfsdk:"delegates"`
+	IncludeEmail         types.Bool   `tfsdk:"include_email"`
+	IdToken              types.String `tfsdk:"id_token"`
+}
+
+func (p *googleEphemeralServiceAccountIdToken) Schema(ctx context.Context, req ephemeral.SchemaRequest, resp *ephemeral.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		Attributes: map[string]schema.Attribute{
```

Comment:
Please update the schema with Description & MarkdownDescription values, similar to [feedback here](https://github.com/GoogleCloudPlatform/magic-modules/pull/12140#pullrequestreview-2434708870).

Reply:
added

Reply:
Sorry, I should have been more explicit. Could you add the per-field descriptions?

Reply:
added as well as included the ephemeral-resources markdown